scripts/release-branch.js

```diff
@@ -82,32 +85,29 @@ fs.writeFileSync(changelogPath, allLogs, (err) => {
 // Checkout the branch.
 const releaseBranch = 'release/' + minor;
 
-git().checkoutLocalBranch(releaseBranch, () => {
-        console.log(`On current release branch: ${releaseBranch}`)
-      })
-     .add('./*')
-     .commit('changelog update and remove old changelog files')
-     .push('origin', releaseBranch);
-
-
-// Create the pull request in GitHub
-const { DANGER_GITHUB_API_TOKEN } = process.env
-
-const octokit = new Octokit({
-  auth: DANGER_GITHUB_API_TOKEN
-});
-
-const pullRequest = {
-  owner: 'massgov',
-  repo: 'mayflower',
-  title: `Release ${minor}`,
-  head: releaseBranch,
-  base: 'master'
-}
-
-octokit.pulls
-  .create(pullRequest)
-  .catch(function() {
-    console.error(`There was an error creating the Github PR: ${e.toString()}`);
-    process.exit(1);
-  })
+(async function() {
+  // This asynchronous logic will happen sequentially.
+  // If an error is thrown, it will break out of this
+  // asynchronous function immediately and exit 1.
+
+  // Create the release branch and push to Github.
+  await git().deleteLocalBranch(releaseBranch)
```

Comment:
I think simple-git wants to use method chaining instead of recalling `git()`
e.g. 

```
git.checkIsRepo()
   .then(isRepo => !isRepo && initialiseRepo(git))
   .then(() => git.fetch());
```

Reply:
Yeah, I just noticed that they recommend doing this:
```
const git = require('simple-git/promise')();
```
I updated the PR accordingly.  We're using `async/await` in place of promises, so these two pieces of code are functionally equivalent :
```js
await git.add('./*');
await git.commit('Changelog update and remove old changelog files');
# vs:
git.add('./*')
  .then(() => git.commit('Changelog update and remove old changelog files'));
```
I just find the await version a lot more readable and harder to mess up.

Reply:
cool, should we test this in circle and try to get ths one in for the release?